### PR TITLE
Update Want-Digest to Want-Repr-Digest [RHELDST-17582]

### PR DESCRIPTION
**Pull request by negillett**

### CODE REVIEW

File: tests/integration/test_exodus.py [hunk 1]
```diff
@@ -134,19 +134,19 @@ def test_content_type_header_HEAD(
 ]
 
 
-# use Want-Digest/Digest to check if alias take effect
+# use Want-Repr-Digest/Repar-Digest to check if alias take effect
```
Comment (rohanpm) [inline tests/integration/test_exodus.py:137]:
```suggestion
# use Want-Repr-Digest/Repr-Digest to check if alias take effect
```

**Review by rohanpm — DISMISSED** [review]
Note that pipeline-test will also need an update for this to get through exodus-cdn-deploy.

**Review by crungehottman — DISMISSED** [review]

**Comment by negillett** [comment]
Will squash

**Comment by negillett** [comment]
> Note that pipeline-test will also need an update for this to get through exodus-cdn-deploy.

Thanks for pointing this out!

**Review by crungehottman — APPROVED** [review]

**Pull request merged**